pallets/handles/README.md

```diff
@@ -0,0 +1,74 @@
+# Handles Pallet
+
+Creates human-readable, homoglyph attack resistent handles for MSA.
+
+## Summary
+
+Provides MSAs with an optional, but unique handle.
+The handles are able to be
+
+- **Base Handle:** The user's chosen handle string. It is not guaranteed to be unique.
```

Comment:
```suggestion
A handle consists of:

- **Canonical Base Handle:** The user's chosen handle string. It is not guaranteed to be unique.
```

Reply:
Canonical is not the user's chosen string. It is derived from it.